### PR TITLE
Remove main conda channel from Nightly and Release

### PR DESCRIPTION
**Pull request by Alexsandruss**
### Description 
Remove main conda channel from Nightly and Release

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
- [ ] The unit tests pass successfully.
- [ ] I have run it locally and tested the changes extensively.
- [x] I have resolved any merge conflicts that might occur with the base branch.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
- [x] I have added a respective label(s) to PR if I have a permission for that.

### CODE REVIEW

**Comment by ethanglaser** [comment]
Would removal of "main channel" section from our docs also be in the scope of this PR? https://intel.github.io/scikit-learn-intelex/latest/installation.html

**Comment by Alexsandruss** [comment]
> Would removal of "main channel" section from our docs also be in the scope of this PR? https://intel.github.io/scikit-learn-intelex/latest/installation.html

Not necessary in this PR, usage in sklearnex CI and sklearnex distribution channels are different things.

**Review by ethanglaser — APPROVED** [review]

**Comment by ethanglaser** [comment]
Assuming no issues with CI

**Review by napetrov — APPROVED** [review]
